### PR TITLE
 Fix #19 - On stop callback for when animation completes

### LINKED ISSUE #19: OnStop callback to notify the app of animation stop

### PR DESCRIPTION
**Pull request by andrewlord1990**
Added callback for onStop when animation stops. This becomes works nicely when combined with #35 (which allows GIFs with a loop count). 

Fixes #19.

### CODE REVIEW

File: app/src/main/java/com/felipecsl/gifimageview/app/MainActivity.java [hunk 1]
```diff
@@ -39,6 +40,16 @@ public Bitmap onFrameAvailable(Bitmap bitmap) {
       }
     });
 
+    gifImageView.setOnAnimationStop(new GifImageView.OnAnimationStop() {
+      @Override public void onAnimationStop() {
+        MainActivity.this.runOnUiThread(new Runnable() {
```
Comment (felipecsl) [inline app/src/main/java/com/felipecsl/gifimageview/app/MainActivity.java]: you can omit `MainActivity.this`

Comment (andrewlord1990) [inline app/src/main/java/com/felipecsl/gifimageview/app/MainActivity.java]: Sure, wasn't sure on the style you would want.

Comment (felipecsl) [inline app/src/main/java/com/felipecsl/gifimageview/app/MainActivity.java]: :+1:

Comment (andrewlord1990) [inline app/src/main/java/com/felipecsl/gifimageview/app/MainActivity.java]: Done.

**Comment by felipecsl** [comment]
:100:

**Pull request merged**